src/EosForm.tsx

```diff
@@ -24,23 +22,26 @@ const EosForm = ({
 
   return (
     <div>
-      <label htmlFor="searchInput" className="searchInput">
-        <input
-          ref={inputField}
-          type="text"
-          id="searchInput"
-          placeholder="Search..."
-          onKeyUp={handleKeyUp}
-        />
-        <div onClick={clearValue}>
-          <EOS_CLOSE_OUTLINED size="sm" />
+      <div className="searchBar">
+        <div style={{ margin: "auto 5 auto 5" }}>
+          <img
+            src="https://img.icons8.com/glyph-neue/24/000000/search--v1.png"
```

Comment:
We can use our own icons right?

Reply:
https://eos-icons.com/?iconName=search&type=static

Reply:
yeah i will make the appropirate changes.

Reply:
Done